cmd/kpod/spec.go

```diff
@@ -91,16 +93,30 @@ func createConfigToOCISpec(config *createConfig) (*spec.Spec, error) {
 		configSpec.Linux.Resources.Pids.Limit = config.resources.pidsLimit
 	}
 
+	// SECURITY OPTS
+	configSpec.Process.NoNewPrivileges = config.noNewPrivileges
+	configSpec.Process.ApparmorProfile = config.apparmorProfile
+	configSpec.Process.SelinuxLabel = config.processLabel
+	configSpec.Linux.MountLabel = config.mountLabel
+	if config.seccompProfilePath != "" && config.seccompProfilePath != "unconfined" {
+		seccompProfile, err := ioutil.ReadFile(config.seccompProfilePath)
+		if err != nil {
+			return nil, errors.Wrapf(err, "opening seccomp profile (%s) failed", config.seccompProfilePath)
+		}
+		var seccompConfig spec.LinuxSeccomp
```

Comment:
I don't think we can do this, Docker uses a different profile struct than runc - extra fields are included

Reply:
It is not blowing up. I believe I grabbed this code from CRI-O?

Reply:
...Huh. If CRI-O uses it, we're probably fine? And if we're not, I'm sure we'll get bug reports later. Fine with leaving this if it works.

Reply:
Well that is the thing, Not sure if anyone has tested this with CRI-O either.  Once I have capabilties merged, I can see if some syscalls are still missing.  Then we will know if this is working or not.  For now it does not blow up when it reads and loads the cri-o/seccomp.json file.

BTW I want to move this out of CRI-O and into skopeo-containers   We should ship this in /etc/containers/seccomp.json and have kpod/cri-o and buildah all share it.

Reply:
@mrunalp FYI

Reply:
Hm. Maybe CRI-O uses a profile in the standard runc format, which would work fine like this. I wonder how this would react to Docker-formatted profiles. Something to think about for compatability.

Reply:
Yes I agree.  Lets open an issue for this, Since this is supposed to handle user input.